packages/ckeditor5-package-generator/lib/utils/choose-package-manager.js

```diff
@@ -6,18 +6,26 @@
 import inquirer from 'inquirer';
 import isYarnInstalled from './is-yarn-installed.js';
 import isPnpmInstalled from './is-pnpm-installed.js';
+import chalk from 'chalk';
 
 /**
+ * @param {Logger} logger
  * @param {Boolean} useNpm
  * @param {Boolean} useYarn
  * @param {Boolean} usePnpm
  * @returns {Promise<'npm'|'yarn'|'pnpm'>}
  */
-export default async function choosePackageManager( useNpm, useYarn, usePnpm ) {
+export default async function choosePackageManager( logger, useNpm, useYarn, usePnpm ) {
```

Comment:
```suggestion
export default async function choosePackageManager( logger, { useNpm, useYarn, usePnpm } ) {
```

Instead of two parameters, could we use two?